analysis_pqc/__init__.py

```diff
@@ -196,6 +197,10 @@ def analyse_cv(v, c, area=1.56e-6, carrier='electrons', cut_param=0.008, savgol_
     #spl_dev = spl(x_norm, 1)
     spl_dev = scipy.signal.savgol_filter(y_norm, window_length=savgol_windowsize, polyorder=1, deriv=1)
 
+    # for definition of fit region, only consider voltages < max_v
+    idv_max=max([i for i,a in enumerate(v) if abs(a)<max_v])
+    spl_dev=spl_dev[:idv_max]
```

Comment:
Hi Veronika, operators and commas should always be surrounded by a single white space to enhance readability. Have a look at Python's coding style guide (PEP8), see https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements